ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -0,0 +1,313 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Workflows
+
+*Workflows > Workflows > Tab OVERVIEW*
+
+![Overview](/Assets/Screenshots/ActindoWorkFlow/Workflows/Workflows.png "[Overview]")
+
+
+- *VIEW*  
+Click the drop-down list to select the view. All created views are displayed in the drop-down list. Click the ![Points](/Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *VIEW* drop-down list to display the context menu and create a view. For detailed information, see [Create view](#create-view).
```

Comment:
Link wird nicht funktionieren, da in PIM. Entweder anpassen oder den letzten Satz mit Link einfach streichen...